### PR TITLE
Remove bootstrap special case alias

### PR DESCRIPTION
**Pull request by YangLyu89**
This is to resolve https://github.com/aspnet/LibraryManager/issues/120

### CODE REVIEW

File: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs [hunk 1]
```diff
@@ -214,7 +214,7 @@ private IEnumerable<CdnjsLibraryGroup> GetSortedSearchResult(string term)
 
         private static string NormalizedGroupName(string groupName)
         {
-            string cleanName = AliasedName(groupName);
+            string cleanName = groupName;
```
Comment (prafullbhosale) [inline src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: No need for this assignment. You can directly use groupName below.

**Comment by prafullbhosale** [comment]
Oh, just realised this PR is against `master` branch, which is being used for 15.8 Preview3. **Do not** merge this to master yet.

File: src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs (hunk 1 above)
Comment (YangLyu89) [reply src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs]: Will do

**Comment by YangLyu89** [comment]
Close this one and merge it to the rel/preview4 branch.

**Pull request merged**